javascript/todo/src/controls.js

```diff
@@ -0,0 +1,21 @@
+export default class Controls {
+  #titles;
+  #dis;
+  constructor(title, discription) {
+    this.#titles = title;
+    this.#dis = discription;
```

Comment:
Try to make your variable names more understandable
recommended => this.description = description;